### PR TITLE
Version 2.0

### PR DESCRIPTION
**Pull request by Incei**
Updated PBPK model and created evaluation plan/report

### CODE REVIEW

File: evaluation/Workflow.R [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+#' createQualificationReport("C:/Software/QualificationRunner9.1.1", maxSimulationsPerCore = 8)
+#' 
+#' # Create a Qualification Report and update Qualification Version Information on title page
+#' versionInfo <- QualificationVersionInfo$new("1.1", "2.2","3.3")
+#' createQualificationReport("C:/Software/QualificationRunner9.1.1", versionInfo = versionInfo)
+#' 
+
+setwd("C:/Digoxin-Model/evaluation/")
+qualificationRunnerFolder <- "C:/OSPQualification/QualificationRunner11.0.138"
+pkSimPortableFolder <- "C:/pk-sim-portable"
```
Comment (Yuri05) [inline evaluation/Workflow.R:31]: remove these lines

Comment (Yuri05) [inline evaluation/Workflow.R]: remove these lines

File: evaluation/Workflow.R [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+  }
+  return(invisible())
+}
+
+createQualificationReport(qualificationRunnerFolder,
+                          pkSimPortableFolder = pkSimPortableFolder,
+                          createWordReport = FALSE,
+                          maxSimulationsPerCore = 3,
+                          versionInfo = NULL,
+                          wordConversionTemplate = NULL)
```
Comment (Yuri05) [inline evaluation/Workflow.R]: remove these lines

**Comment by Yuri05** [comment]
Fix invalid cross references (used multiple times):

```
  error: bad link report.html#-references
  error: bad link report.html#5-references
  error: bad link report.html#RANGE!5-references

  Found 0 bad links
  Found 45 bad anchors
```

File: evaluation/Input/Content/titlepage.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+
+
+
+
+| Version                                         | 1.1-OSP9.1                                                   |
+| ----------------------------------------------- | ------------------------------------------------------------ |
+| based on *Model Snapshot* and *Evaluation Plan* | https://github.com/Open-Systems-Pharmacology/Digoxin-Model/releases/tag/v1.1 |
+| OSP Version                                     | 11.1                                                         |
+| Qualification Framework Version                 | 2.2                                                          |
```
Comment (Yuri05) [inline evaluation/Input/Content/titlepage.md]: Replace all versions and replace them with placeholders:
compare e.g. https://github.com/Open-Systems-Pharmacology/Alfentanil-Model/blob/master/Evaluation/Input/Content/titlepage.md#building-and-evaluation-of-a-pbpk-model-for-alfentanil-in-adults 

grafik

The versions must be set in the call of `createQualificationReport `(in the versionInfo object - s. Template)

**Comment by Yuri05** [comment]
some of value origins are set to unknown. is it possible to set them to appropriate values right now?
If not, I will create a separate issue for this:


<!DOCTYPE HTML  PUBLIC "-//W3C//DTD HTML 4.0  Transitional//EN">
<html><body>

Name | Top Container | Organ | Molecule | Value | Value Origin
-- | -- | -- | -- | -- | --
EHC continuous fraction | Organism | Liver |   | 1.000 | Unknown
t1/2 (liver) |   |   | CYP3A4 | 36.000 h | Unknown
Reference concentration |   |   | ATP1A2 | 0.477 µmol/l | Unknown
Reference concentration |   |   | ABCB1 | 1.410 µmol/l | Unknown
kcat | Digoxin | ABCB1-Stephens (2001) | Digoxin | 71.163 1/min | Unknown-correctes as 100.34/1.41
Specific clearance | Digoxin-Total Hepatic Clearance-Fitted |   |   | 0.038 1/min | Unknown


</body></html>

**Comment by Yuri05** [comment]
https://github.com/Yuri05/Digoxin-Model/blob/review/evaluation/report/report.md#figure-3-41 
No observed data in this figure?

**Comment by Incei** [comment]
Dear @Yuri05 many thanks for your comments, I will correct these!

**Comment by AndreDlm** [comment]
Additional comments can be found in the attached report. Some of them have already been addressed above...
[report.pdf](https://github.com/Open-Systems-Pharmacology/Digoxin-Model/files/10578601/report.pdf)

**Comment by AndreDlm** [comment]
...and one last comment: Figure 3-27 shows data by Kirby et al. 2012, but this study is not listed in the table in section `2.2.2 Clinical data`

File: evaluation/Input/Evaluation_plan.json [hunk 4]
```diff
@@ -0,0 +1,638 @@
...
+            },
+            {
+                "Project": "Digoxin",
+                "Simulation": "Digoxin po MD, 2x 0.25 mg - 2",
+                "SectionReference": "ct-profiles-model-building"
+            }
+        ],
+        "GOFMergedPlots": [
+            {
+                "Title": "Goodness of fit plor for concentration in plasma.",
```
Comment (Yuri05) [inline evaluation/Input/Evaluation_plan.json]: Typo: **plor**

**Comment by Yuri05** [comment]
Remaining "Unknown" value origins

<!DOCTYPE HTML  PUBLIC "-//W3C//DTD HTML 4.0  Transitional//EN">
<html><body>

Name | Organ | Molecule | Value | Value Origin
-- | -- | -- | -- | --
EHC continuous fraction | Liver |   | 1.000 | Unknown
t1/2 (liver) |   | CYP3A4 | 36.000 h | Unknown
Reference concentration |   | ATP1A2 | 0.391 µmol/l | Unknown
Reference concentration |   | ABCB1 | 1.410 µmol/l | Unknown


</body></html>

**Comment by Yuri05** [comment]
Solubility name in the Compound-BB is interesting :)
grafik

File: evaluation/Input/Content/References.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+**Gurley 2008** Gurley, B.J., Swain, A., Williams, D.K., Barone, G. & Battu, S.K. Gauging the clinical significance of P-glycoprotein-mediated herb-drug interactions: comparative effects of St. John’s wort, Echinacea, clarithromycin, and rifampin on digoxin pharmacokinetics. Mol. Nutr. food Res. 52, 772–9 (2008). 
+
+**Hanke 2018** Hanke N, Frechen S, Moj D, Britz H, Eissing T, Wendl T, Lehr T. PBPK Models for CYP3A4 and P-gp DDI Prediction: A Modeling Network of Rifampicin, Itraconazole, Clarithromycin, Midazolam, Alfentanil, and Digoxin. CPT Pharmacometrics Syst Pharmacol.. 2018 Oct;7(10):647-659.
+
+**Hayward 1978** Hayward, R.P., Greenwood, H. & Hamer, J. Comparison of digoxin and medigoxin in normal subjects. Br. J. Clin. Pharmacol. 6, 81–6 (1978). 
+
+**Hinderling 1984** Hinderling, P.H. Kinetics of partitioning and binding of digoxin and its analogues in the subcompartments of blood. J. Pharm. Sci. 73, 1042–53 (1984).
+
+**Jalava 1997 ** Jalava, K.M., Partanen, J. & Neuvonen, P.J. Itraconazole decreases renal clearance of digoxin. Ther. Drug Monit. 19, 609–13 (1997).
```
Comment (Yuri05) [inline evaluation/Input/Content/References.md]: Remove space before `** `at the end to show properly

**Review by Yuri05 — APPROVED** [review]

**Pull request merged**